ios/MaasCore/Package.swift

```diff
@@ -36,11 +36,11 @@ let package = Package(
              path: "Sources/CoreBinary/CoreBinary.xcframework"),
         .target(
             name: "MaasCore",
-            dependencies: ["CoreBinary", "MaasTheme", "MaasComponents"],
+            dependencies: ["CoreBinary"],
```

Comment:
OK, this is an important dependency change.
Before, MaasCore included everything.
```
MaasCore > MaasComponents > MaasTheme
```
Now, everything includes MaasCore.
```
MaasComponents > MaasTheme > MaasCore
```

Reply:
I think this makes sense, matches Android, too, where `ui-android` (which has both the theme and components) depends on `:common:core`.

Reply:
Nice!